bitsail-connectors/connector-hbase/pom.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2022 Bytedance Ltd. and/or its affiliates.
```

Comment:
nit: change from 2022 to 2022-2023. Applying on all places as well.

Reply:
Changed it to 2022-2023 based on suggestion.